### ISSUE TITLE
Description is hard to understand and contains a mistake

### ISSUE THREAD

**Issue by alexandreduf** [OCA/account-invoicing#780]
Hi everybody,

The description should be improved. Right now, it's hard to know what the module does and what's its use case.
There is also a mistake in the description ("no" instead of "not")

"This module add a warning popup on invoice to ensure warning is populated no**[t]** only when partner is changed."

What we can understand : 

- Normally, in Vanilla Odoo there is a pop-up when partner is changed.
- This module adds a pop-up for some other events.

What we cannot understand : 

- What are the other events where a pop-up will appear?

Thank you guys for your great work! I love this community!

**Comment by pedrobaeza** [comment]
Which module? Which version?

**Comment by alexandreduf** [comment]
Sorry for that, I'm new to Github.
Thought my issue would be linked to the module that I was previously viewing before switching to "Issues" tab.

Here's the info : 
account_invoice_warn_message/
Version 13

**Comment by pedrobaeza** [comment]
@HviorForgeFlow can you take a look?

**Comment by HviorForgeFlow** [comment]
hi @alexandreduf ,yes.. description can lead to misunderstands. Have you test the module? Maybe adding additional information like that could help:

```
This module add a warning popup on invoice to ensure warning is populated
not only when partner is changed while invoice is in draft.

On standard Odoo, warning messages on Invoices only appear when partner is
changed, this can lead the end user not to see warning messages in some scenarios.
(for example, Invoice automatically created from a Sales Order)
```
This is only a proposal, could you propose a proper description to that module that fit your expectations?

Please open a PR, I will review thankfully.

Thanks,

**Comment by alexandreduf** [comment]
Hi @HviorForgeFlow,

Sorry for the delay.

Here's how I would rephrase this :

> This module add a warning popup on invoice when the following events occur :
> - (Describe event 1)
> - (Describe event 2)
> - (Describe event 3)
> - (Describe event 4)
> - (...)
> 
> On standard Odoo, warning messages on Invoices only appear when partner is changed. Some relevant warnings are missing and are added with this module.

I cannot test the module since I can't read code (my knowledge in coding is very weak) and since the description doesn't help me understanding what the module does exactly. So I have no idea what to test.

Can someone help completing the description so I can test the module and give some feedback?

Regards,
